libs/src/api/track.js

```diff
@@ -341,12 +341,27 @@ class Track extends Base {
         metadataFileUUID,
         transcodedTrackUUID,
         transcodedTrackCID
-      } = await this.creatorNode.uploadTrackContent(
-        trackFile,
-        coverArtFile,
-        metadata,
-        onProgress
-      )
+      } = await retry(async (bail, num) => {
+        return this.creatorNode.uploadTrackContent(
+          trackFile,
+          coverArtFile,
+          metadata,
+          onProgress
+        )
+      }, {
+        // Retry function 3x
+        // 1st retry delay = 500ms, 2nd = 1500ms, 3rd...nth retry = 4000 ms (capped)
+        minTimeout: 500,
+        maxTimeout: 4000,
+        factor: 3,
+        retries: 3,
+        onRetry: (err, i) => {
+          if (err) {
+            // eslint-disable-next-line no-console
+            console.log('Retry error : ', err)
```

Comment:
can we make this log more specific, like `uploadTrackContent Retry error:`

Reply:
I noticed when we get the retry error logs in ci that they're hard to debug what they are unless we do further digging

Reply:
done. FWIW, the stack trace should give us this info